components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -286,14 +327,23 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
             return buttonText;
         }
 
-        const lastStep = this.getLastStep(TutorialSteps[category]);
+        const lastStep = this.getLastStep(TutorialSteps[category], category);
         if (this.props.step === lastStep) {
-            buttonText = (
-                <FormattedMessage
-                    id={'tutorial_tip.finish_tour'}
-                    defaultMessage={'Finish tour'}
-                />
-            );
+            if (this.props.customLastStepButtonText) {
+                buttonText = (
+                    <FormattedMessage
+                        id={this.props.customLastStepButtonText.id}
```

Comment:
will this work well with the `make i18n-extract` action. I mean, if someone runs that command later won't it delete whatever value you have set to en.json?

Reply:
@pablovelezvidal 3/5 it wouldn't affect. Have you seen this happen before?

Reply:
it happened to me, but don't remember if it is the same exact situation. You can check it if you run the command and verify if it deletes the entry you added in the  en.json file

Reply:
I already ran it